samples/Diagnostics.StatusCodes.Mvc/Controllers/ErrorsController.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNet.Mvc;
+
+namespace Diagnostics.StatusCodes.Mvc.Controllers
+{
+    public class ErrorsController : Controller
```

Comment:
Should we litter a few comments around here to say what this is? I'll proposed some comments:

`// This controller is called to handle application errors`